Source/WebKit/WebProcess/WebPage/FindController.cpp

```diff
@@ -78,7 +78,8 @@ FindController::~FindController()
 
 PluginView* FindController::mainFramePlugIn()
 {
-    return m_webPage->mainFramePlugIn();
+    RefPtr protectedWebPage { m_webPage.get() };
```

Comment:
We do not use `protected` naming for local variables in new code. This should just be `webPage`